store/store_test.go

```diff
@@ -75,19 +75,19 @@ func TestBlockHash(t *testing.T) {
 
 	sb, _ := td.store.Block(1)
 
-	assert.Equal(t, hash.UndefHash, td.store.BlockHash(0))
-	assert.Equal(t, hash.UndefHash, td.store.BlockHash(util.MaxUint32))
-	assert.Equal(t, sb.BlockHash, td.store.BlockHash(1))
+	assert.Equal(t, td.store.BlockHash(0), hash.UndefHash)
```

Comment:
Expected value should come first. So please revert this change.

Reply:
Yes, my branch was outdated. Already fixed. Thanks for the warning 🙂